raiden_contracts/deploy/etherscan_verify.py

```diff
@@ -53,6 +51,11 @@ def etherscan_verify(chain_id, apikey, guid, contract_name):
         etherscan_api = 'https://api-kovan.etherscan.io/api'
     elif chain_id == 1:
         etherscan_api = 'https://api.etherscan.io/api'
+    else:
+        click.echo(f"Unknown chain_id {chain_id}", err=True)
```

Comment:
please, consider adopting the same code style we use in the raiden code base.

Reply:
I'll have a look how it's done there.

Reply:
I suddenly realize this line has two arguments.